modules/engine/src/main/scala/observe/engine/Engine.scala

```diff
@@ -140,67 +140,47 @@ class Engine[F[_]: MonadThrow: Logger, S, U] private (
       _.map { seq =>
         seq.status match {
           case SequenceState.Running(userStop, internalStop, _, _, _) =>
-            if (userStop || internalStop) {
```

Comment:
it seems we don't care about `userStop` anymore?

Reply:
`userStop` seems to be true when the user requests a pause after the current step completes.